### PR TITLE
sing in + launch postman in navbar

### PR DESCRIPTION
**Pull request by ch264**
added GA tracker to LC sign in button and changed Dashboard to Launch + GA

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -31,13 +32,52 @@ const searchClient = {
...
   const { cookie } = props;
+
   if (cookie !== 'yes') {
     return (
-      <a href="https://identity.getpostman.com/login" className="btn btn__primary">Sign In</a>
+      <a
+        href="https://identity.getpostman.com/login"
+        className="btn btn__primary"
+        target="_blank"
+        onClick={(e) => {
```
Comment (oswish) [inline src/components/Header/Header.jsx:42]: L42-no-e

I think this^ is just a matter of removing "e" & having it just me an empty "()".

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -31,13 +32,52 @@ const searchClient = {
...
+      </a>
     );
   }
   return (
-    <a href="https://app.getpostman.com" className="btn btn__primary">Dashboard</a>
+    <a
+      href="https://go.postman.co/build"
+      className="btn btn__primary"
+      target="_blank"
+      onClick={(e) => {
```
Comment (oswish) [inline src/components/Header/Header.jsx:65]: l65-no-e

_Same as [L42](https://github.com/postmanlabs/postman-docs/pull/2618/files#diff-da78a44fef431db9a14bc689b72b9411R42)_; think you just remove "e" for an empty "()".

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -31,13 +32,52 @@ const searchClient = {
...
 const LoginCheck = (props) => {
   const { cookie } = props;
+
   if (cookie !== 'yes') {
     return (
-      <a href="https://identity.getpostman.com/login" className="btn btn__primary">Sign In</a>
+      <a
+        href="https://identity.getpostman.com/login"
+        className="btn btn__primary"
+        target="_blank"
```
Comment (oswish) [inline src/components/Header/Header.jsx:41]:
```sh
41:9   error  Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener  react/jsx-no-target-blank
```

I think it's just a matter of adding `rel="noreferrer"`.

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -31,13 +32,52 @@ const searchClient = {
...
+        Sign In
+      </a>
     );
   }
   return (
-    <a href="https://app.getpostman.com" className="btn btn__primary">Dashboard</a>
+    <a
+      href="https://go.postman.co/build"
+      className="btn btn__primary"
+      target="_blank"
```
Comment (oswish) [inline src/components/Header/Header.jsx:64]:
```sh
64:7   error  Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener  react/jsx-no-target-blank
```

_Same as [L42](https://github.com/postmanlabs/postman-docs/pull/2618/files#diff-da78a44fef431db9a14bc689b72b9411R41)_, it's a matter of adding `rel="noreferrer"`.

**Comment by ch264** [comment]
Screen Shot 2020-09-01 at 1 00 00 PM


Thanks @oswish I fixed the linter errors and when a user is logged in they see the 'Launch Postman button'.

I think that we need this in develop to test the GA integration

**Pull request merged**